rust/Cargo.toml

```diff
@@ -21,17 +21,19 @@ bytes = "0.5.3"
 log = "0"
 regex = "1"
 chrono = "0"
-crossbeam = "0"
 rusoto_core = "0.43"
 rusoto_s3 = "0.43"
+pyo3 = { version = "0", features = ["extension-module"], optional = true }
```

Comment:
Why does this dependency need to come into the root crate? I'm sure there's a reason but would you please comment in the file why this exists here? (cause it feels leaky)